python/semantic_kernel/kernel.py

```diff
@@ -727,9 +727,11 @@ def _create_semantic_function(
         if function_config.has_chat_prompt:
             service = self.get_ai_service(
                 ChatCompletionClientBase,
-                function_config.prompt_template_config.default_services[0]
-                if len(function_config.prompt_template_config.default_services) > 0
-                else None,
+                (
```

Comment:
This is old code -- please update your local repo to latest main. This code hasn't been present since 0.5.1.dev0. We're now on 0.9.4b1.